### PR TITLE
Add support for Ubuntu

### PR DESCRIPTION
**Pull request by nisanharamati**
Tested with 18.04 Bionic

I couldn't get bork to work with some of the commands, so I dropped to an all bash install script for Ubuntu, and a case statement in the main install.sh script to choose between bork for OSX and shell scripts for Ubuntu.

### CODE REVIEW

**Comment by robzienert** [comment]
@nisanharamati wowowow. Awesome. Sadly, you're gonna have to rebase - there's conflicts now.

**Comment by nisanharamati** [comment]
@robzienert Rebased and tested
@cfieber could you take a look as well to make sure I didn't mess any of your changes by accident?

File: satisfy/linux.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+apt install -y redis
+apt install -y mysql-server-5.7
+
+# install nvm
+mkdir -p $HOME/.nvm
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+source ~/.bashrc
+
+# Install yarn
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
```
Comment (cfieber) [inline satisfy/linux.sh:17]: There's a chance that the yarn from apt is too old for the deck codebase

Was why I switched the MacOS script away from brew install yarn to npm -g install yarn

But, if the UI doesn't complain then sticking with the apt is probably best

**Review by cfieber — APPROVED** [review]
looks good!

File: satisfy/linux.sh (hunk 1 above)
Comment (nisanharamati) [reply satisfy/linux.sh:17]: That's a good point. Since we still have the `npm -g install yarn` later, I think it should be safe to remove this part?

Comment (nisanharamati) [reply satisfy/linux.sh:17]: At least for now it looks like apt has the latest version, so I'm going to leave it in there.

**Pull request merged**